apps/dashboard/src/contract-ui/tabs/nfts/page.tsx

```diff
@@ -48,6 +49,18 @@ export const ContractNFTPage: React.FC<NftOverviewPageProps> = ({
     "ERC721ClaimCustom",
   ]);
 
+  const detectedClaimState = extensionDetectedState({
```

Comment:
nice, I like moving this up - we also have to figure out how to do this same thing in v5 soon